tests/inputFields.spec.ts

```diff
@@ -11,46 +11,46 @@ test.describe('Pet types', () => {
     })
 
     test('Update pet type', async({page}) => {
-        const petTypesTableName = page.getByRole('heading', {name: "Pet Types"})
+        const petTypesTableName = page.locator('app-pettype-list h2')
         await expect(petTypesTableName).toHaveText('Pet Types')  //2. assertion of Pet Types text displayed
 
-        await page.getByRole('button', {name: "Edit"}).first().click()  //3. click on edit button for cat
+        const petTypeTableEditButton = page.getByRole('button', {name: "Edit"})
+        await petTypeTableEditButton.first().click()  //3. click on edit button for cat
         await expect(page.getByRole('heading', {name: "Edit Pet Type"})).toBeVisible() //4. assertion of Edit Pet Type header
 
-        const editPetTypeName = page.locator('#name')
-        await editPetTypeName.click()
-        await editPetTypeName.clear()
-        await editPetTypeName.fill('rabbit') //5. change pet name to rabbit and click update
-        const updatedPetTypeNameValue = await editPetTypeName.inputValue()
-        expect(updatedPetTypeNameValue).toEqual('rabbit')
+        const editPetTypeNameInputField = page.locator('#name')
+        await editPetTypeNameInputField.click()
+        await editPetTypeNameInputField.clear()
+        await editPetTypeNameInputField.fill('rabbit') //5. change pet name to rabbit and click update
+        expect(editPetTypeNameInputField).toHaveValue('rabbit')
```

Comment:
await is missing.
Locator assertion always should have "await" otherwise it will not auto wait for the desired condition